### PR TITLE
Temporarily disable content indexing.

### PR DESCRIPTION
**Pull request by robdodson**
Related to #3836 

Changes proposed in this pull request:

- Temporarily disable content indexing API. We should turn this back on when the underlying bug is fixed in the API itself.

### CODE REVIEW

**Review by MichaelSolati — APPROVED** [review]

File: src/lib/app.js [hunk 1]
```diff
@@ -8,14 +8,16 @@
...
 import {swapContent, getHTML} from './loader';
 import * as router from './utils/router';
 import {checkUserPreferredLanguage} from './actions';
 import {store} from './store';
 import {localStorage} from './utils/storage';
 import removeServiceWorkers from './utils/sw-remove';
-import {syncContentIndex} from './content-indexing';
+// TODO: Enable this when #3836 is fixed.
+// // https://github.com/GoogleChrome/web.dev/issues/3836
+// import {trackError} from './analytics'; // side effects & named export
```
Comment (jeffposnick) [inline src/lib/app.js:19]: I think you want to keep this? Or at least double-check what the side-effects of the import is. I remember that the `./analytics` import was around before my PR.

**Review by jeffposnick — CHANGES_REQUESTED** [review]

File: src/lib/app.js (hunk 1 above)
Comment (robdodson) [reply src/lib/app.js:19]: It looks like it's ok being commented out because your snippet on line 70 was the only one using it.

Comment (robdodson) [reply src/lib/app.js:19]: OH WAIT you're right

Comment (robdodson) [reply src/lib/app.js:19]: didn't realize that module had its own side effects. Good catch.

**Pull request merged**